examples/dtls-sock/Makefile

```diff
@@ -0,0 +1,67 @@
+# name of your application
+APPLICATION = dtls_sock
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# TinyDTLS only has support for 32-bit architectures ATM
+BOARD_BLACKLIST := arduino-duemilanove arduino-mega2560 arduino-uno chronos \
```

Comment:
Please use `FEATURES_REQUIRED += arch_32bit` as a dependency for `tinydtls` (this is new ;-)).

Reply:
Replaced `BOARD_BLACKLIST` with `FEATURES_REQUIRED`. Got warning about features not supported but rebasing to latest master solved it. Should I also move `BOARD_INSUFFICIENT_MEMORY` to `Makefile.ci ` like in #12406?

Reply:
Yeah, otherwise the CI will now complain ;-).

Reply:
Done.